src/pages/ContentScripts/Perceptor.tsx

```diff
@@ -60,7 +60,7 @@ export class Perceptor extends PerceptorBase {
         logger.info(featureId, 'is disabled');
         return;
       }
-      if (Feature.prototype.include.every((c: () => any) => !c())) {
+      if (!Feature.prototype.include.every((c: () => any) => c())) {
```

Comment:
I think this way is more natural, too :)

Reply:
The logic is different here. The previous version implemented an `or` logic and in this change it is an `and` logic.

Reply:
Ah oh, that is not right. Take an example:
<img width="1548" alt="image" src="https://user-images.githubusercontent.com/32434520/161387228-00274f33-2d53-4c26-8925-d70ae5bf438f.png">
The `Hypertrons` is the button injected in both PR & Issue pages:
<img width="990" alt="image" src="https://user-images.githubusercontent.com/32434520/161387269-01bfd87e-5c9e-405c-b3dd-3b9fd261d9ab.png">

So it should be `or` rather `and`. Maybe another strategy should be made to handle this.

Reply:
OK, I will try to figure!